components/sup/src/main.rs

```diff
@@ -358,53 +358,32 @@ fn sub_status(m: &ArgMatches) -> Result<()> {
         sup::output::set_no_color(true);
     }
     let cfg = mgrcfg_from_matches(m)?;
-    let mut services: Vec<ServiceStatus> = Vec::new();
-
     if !Manager::is_running(&cfg)? {
-        outputln!("The supervisor is not running.");
+        println!("The supervisor is not running.");
         std::process::exit(3);
     }
-
     match m.value_of("PKG_IDENT") {
         Some(pkg) => {
             match Manager::service_status(cfg, PackageIdent::from_str(pkg)?) {
-                Ok(status) => {
-                    services.push(status);
-                }
+                Ok(status) => outputln!("{}", status),
                 Err(_) => {
-                    outputln!("The {} service is not currently loaded.", pkg);
+                    println!("{} is not currently loaded.", pkg);
                     std::process::exit(2);
                 }
             }
         }
-        None => services.append(&mut Manager::status(cfg)?),
-    }
-
-    for status in services {
-        match status.process.state {
-            ProcessState::Up => {
-                outputln!("The {} service entered the running state with PID: {} {} seconds ago.",
-                          status.pkg.ident,
-                          status.process.pid.unwrap(),
-                          status.process.elapsed.num_seconds())
-            }
-            ProcessState::Down => {
-                outputln!("The {} service is not currently running.", status.pkg.ident);
-            }
-            ProcessState::Start => {
-                outputln!("The {} service entered the starting state {} seconds ago.",
-                          status.pkg.ident,
-                          status.process.elapsed.num_seconds());
+        None => {
+            let services = Manager::status(cfg)?;
+            if services.is_empty() {
+                println!("No services loaded.");
+                return Ok(());
             }
-            ProcessState::Restart => {
-                outputln!("The {} service entered the restarting state {} seconds ago.",
-                          status.pkg.ident,
-                          status.process.elapsed.num_seconds());
+            for status in services {
+                println!("{}", status);
```

Comment:
I did intentionall change these from `outputln!` to `println!`. The intent of `outputln!` was to let us know which module a log message was coming from and if it was from a service or from the supervisor itself. That information isn't very useful when we're just dealing with one-off commands like this